datalad/support/gitrepo.py

```diff
@@ -195,15 +196,18 @@ def _normalize_path(base_dir, path):
     # BUT with relative curdir/pardir start it would assume relative to curdir
     #
     elif path.startswith(_curdirsep) or path.startswith(_pardirsep):
-        path = normpath(opj(realpath(getpwd()), path))  # realpath OK
+        # XXX use of normpath() likely problematic
+        path = normpath(
```

Comment:
again, `normpath` isn't only problematic, but superfluous in combination with `resolve()`.
Which would then lead to `path = str((Path(getpwd()) / pathobj).resolve())` instead, I think.

Reply:
Cool, will remove.

Reply:
Done